components/localized-msg.jsx

```diff
@@ -0,0 +1,15 @@
+var React = require('react/addons');
```

Comment:
Ideally we should but I can't figure out out why I need to require mixins
twice yet
On Jun 19, 2015 9:15 PM, "ScottDowne" <notifications@github.com> wrote:

> In components/localized-msg.jsx
> <https://github.com/mozilla/donate.mozilla.org/pull/49#discussion_r32880537>
> :
>
> > @@ -0,0 +1,15 @@
> > +var React = require('react/addons');
>
> We are not using localized-msg.jsx anymore are we?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/donate.mozilla.org/pull/49/files#r32880537>.
>